examples/keda-scale-to-zero/create-claim.py

```
@@ -0,0 +1,94 @@
+
+# Configuration
+NAMESPACE = "keda-test"
+WARMPOOL = "python-sdk-warmpool"
+RATE_PER_SECOND = 5
+TEST_DURATION_MINUTES = 10
+CLAIM_TTL_SECONDS = 60
+
+# Initialize K8s Client
+try:
+    config.load_kube_config()
+except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)